pylint/lint/run.py

```diff
@@ -72,13 +73,11 @@ class Run:
     def _return_one(*args):  # pylint: disable=unused-argument
         return 1
 
-    def __init__(
+    def initialize(
         self,
-        args,
+        args: list,
```

Comment:
```suggestion
        args: List,
```
Unless we import the typing from future :)